File: app/src/main/java/com/example/annada/eventfinder/model/SuggestedBounds.java
package com.example.annada.eventfinder.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by : annada
 * Date : 21/10/2017.
 */

public class SuggestedBounds {
    @SerializedName("ne")
    @Expose
    private Ne_ ne;
    @SerializedName("sw")
    @Expose
    private Sw_ sw;

    public Ne_ getNe() {
        return ne;
    }

    public void setNe(Ne_ ne) {
        this.ne = ne;
    }

    public Sw_ getSw() {
        return sw;
    }

    public void setSw(Sw_ sw) {
        this.sw = sw;
    }
}

File: app/src/main/java/com/example/annada/eventfinder/model/SuggestedBounds.kt
package com.example.annada.eventfinder.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by : annada
 * Date : 21/10/2017.
 */

class SuggestedBounds {
    @SerializedName("ne")
    @Expose
    var ne: Ne_? = null
    @SerializedName("sw")
    @Expose
    var sw: Sw_? = null
}